k8s-tester/csi-ebs/tester.go

```diff
@@ -13,52 +12,77 @@ import (
 	"time"
 
 	"github.com/aws/aws-k8s-tester/client"
-	"github.com/aws/aws-k8s-tester/k8s-tester/helm"
+	helm "github.com/aws/aws-k8s-tester/k8s-tester/helm"
 	k8s_tester "github.com/aws/aws-k8s-tester/k8s-tester/tester"
+	"github.com/aws/aws-k8s-tester/utils/rand"
+	utils_time "github.com/aws/aws-k8s-tester/utils/time"
 	"github.com/manifoldco/promptui"
 	"go.uber.org/zap"
+	core_v1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	storage_v1 "k8s.io/api/storage/v1"
+	k8s_errors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	api_resource "k8s.io/apimachinery/pkg/api/resource"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/utils/exec"
 )
 
 type Config struct {
 	Enable bool `json:"enable"`
 	Prompt bool `json:"-"`
 
-	Stopc     chan struct{} `json:"-"`
 	Logger    *zap.Logger   `json:"-"`
 	LogWriter io.Writer     `json:"-"`
+	Stopc     chan struct{} `json:"-"`
 	Client    client.Client `json:"-"`
 
 	// MinimumNodes is the minimum number of Kubernetes nodes required for installing this addon.
-	MinimumNodes int `json:"minimum_nodes"`
-	// HelmChartRepoURL is the helm chart repo URL.
-	HelmChartRepoURL string `json:"helm_chart_repo_url"`
+	MinimumNodes int `json:"minimum-nodes"`
+	// Namespace to create test resources.
+	Namespace string `json:"namespace"`
 }
 
 func (cfg *Config) ValidateAndSetDefaults() error {
-
+	if cfg.MinimumNodes == 0 {
+		cfg.MinimumNodes = DefaultMinimumNodes
+	}
+	if cfg.Namespace == "" {
+		return errors.New("empty Namespace")
+	}
 	return nil
 }
 
-const chartName = "aws-ebs-csi-driver"
-
 const (
-	DefaultMinimumNodes int = 1
-	// https://github.com/kubernetes-sigs/aws-ebs-csi-driver#deploy-driver
-	// https://github.com/kubernetes-sigs/aws-ebs-csi-driver/releases
-	//
-	// TODO: not working
-	// https://github.com/kubernetes-sigs/aws-ebs-csi-driver/releases/download/v0.5.0/helm-chart.tgz
-	// "gzip: invalid header"
-	DefaultHelmChartRepoURL = "https://kubernetes-sigs.github.io/aws-ebs-csi-driver"
```

Comment:
Can we keep this as default? Sometimes, we want to test different CSI EBS drivers?